pkg/operator/status.go

```diff
@@ -279,6 +279,15 @@ func (optr *Operator) syncUpgradeableStatus(co *configv1.ClusterOperator) error
 		Reason: asExpectedReason,
 	}
 
+	configNode, err := optr.nodeClusterLister.Get(ctrlcommon.ClusterNodeInstanceName)
+	if err != nil {
+		return err
+	}
+	if configNode.Spec.CgroupMode == configv1.CgroupModeV1 {
+		coStatusCondition.Status = configv1.ConditionFalse
+		coStatusCondition.Reason = "ClusterOnCgroupV1"
+		coStatusCondition.Message = "Cluster is using deprecated cgroup v1 and is not upgradable. Please update the `CgroupMode` in the `nodes.config.openshift.io` object to 'v2'. Once upgraded, the cluster cannot be changed back to cgroup v1"
```

Comment:
From the slack thread, maybe we can update this message? It should be fine to do so since we're planning to remove this in the 4.19 branch (and thereby have a diff)

Reply:
Updated, Could you approve it?